packages/template/README.md

```diff
@@ -7,7 +7,4 @@ Firebase JS SDK. It will give you a couple things OOTB:
   with the rest of the SDK.
 - **Isomorphic Testing/Coverage:** Your tests will be run in both Node.js and
   Browser environments and coverage from both, collected.
-- **Links to all of the other packages:** Should your new package need to take
```

Comment:
This seems useful and should be kept.

Reply:
It's not true, though, they're not included in `package.json` and they probably shouldn't be because the versions would have to be constantly updated?

It does have app and app-types as peerDependencies but that's all. I looked at the original commit where this line was first written (https://github.com/firebase/firebase-js-sdk/commit/9fcdd378fdd9253248bb385ce87161de64b54592) and it doesn't even include `@firebase/util` then, or maybe I'm confused about what he's saying.

Reply:
Thanks for digging into it. I'm fine as is.